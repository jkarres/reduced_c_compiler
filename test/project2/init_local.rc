// testing local variable initialization

function : int i() { return 42; }

function : float f() { return 9.8; }

function : bool b() { return false; }

function : void main() {
  int ii = i(), iii = i() + i();

  float ff = f();

  bool bb = b();


  cout << 1 << " " << ii << endl;
  cout << 2 << " " << iii << endl;
  cout << 3 << " " << ff << endl;
  cout << 4 << " " << bb << endl;

}
