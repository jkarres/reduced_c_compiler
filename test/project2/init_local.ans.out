1 42
2 84
3 9.80
4 false
